Cannot compile on GHC 8.10

```
Building library for world-peace-1.0.1.0..
[1 of 5] Compiling Data.WorldPeace.Internal.Prism ( src/Data/WorldPeace/Internal/Prism.hs, dist/build/Data/WorldPeace/Internal/Prism.o, dist/build/Data/WorldPeace/Internal/Prism.dyn_o )
[2 of 5] Compiling Data.WorldPeace.Internal ( src/Data/WorldPeace/Internal.hs, dist/build/Data/WorldPeace/Internal.o, dist/build/Data/WorldPeace/Internal.dyn_o )
[3 of 5] Compiling Data.WorldPeace.Product ( src/Data/WorldPeace/Product.hs, dist/build/Data/WorldPeace/Product.o, dist/build/Data/WorldPeace/Product.dyn_o )

src/Data/WorldPeace/Product.hs:56:23: error:
    Not in scope: type variable ‘u’
   |
56 | instance forall (f :: u -> *) (a :: u). ToProduct (f a) f '[a] where
   |                       ^

src/Data/WorldPeace/Product.hs:56:37: error:
    Not in scope: type variable ‘u’
   |
56 | instance forall (f :: u -> *) (a :: u). ToProduct (f a) f '[a] where
   |                                     ^

src/Data/WorldPeace/Product.hs:61:23: error:
    Not in scope: type variable ‘u’
   |
61 | instance forall (f :: u -> *) (a :: u) (b :: u). ToProduct (f a, f b) f '[a, b] where
   |                       ^

src/Data/WorldPeace/Product.hs:61:37: error:
    Not in scope: type variable ‘u’
   |
61 | instance forall (f :: u -> *) (a :: u) (b :: u). ToProduct (f a, f b) f '[a, b] where
   |                                     ^

src/Data/WorldPeace/Product.hs:61:46: error:
    Not in scope: type variable ‘u’
   |
61 | instance forall (f :: u -> *) (a :: u) (b :: u). ToProduct (f a, f b) f '[a, b] where
   |                                              ^

src/Data/WorldPeace/Product.hs:66:23: error:
    Not in scope: type variable ‘u’
   |
66 | instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u). ToProduct (f a, f b, f c) f '[a, b, c] where
   |                       ^

src/Data/WorldPeace/Product.hs:66:37: error:
    Not in scope: type variable ‘u’
   |
66 | instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u). ToProduct (f a, f b, f c) f '[a, b, c] where
   |                                     ^

src/Data/WorldPeace/Product.hs:66:46: error:
    Not in scope: type variable ‘u’
   |
66 | instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u). ToProduct (f a, f b, f c) f '[a, b, c] where
   |                                              ^

src/Data/WorldPeace/Product.hs:66:55: error:
    Not in scope: type variable ‘u’
   |
66 | instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u). ToProduct (f a, f b, f c) f '[a, b, c] where
   |                                                       ^

src/Data/WorldPeace/Product.hs:71:23: error:
    Not in scope: type variable ‘u’
   |
71 | instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u) (d :: u). ToProduct (f a, f b, f c, f d) f '[a, b, c, d] where
   |                       ^

src/Data/WorldPeace/Product.hs:71:37: error:
    Not in scope: type variable ‘u’
   |
71 | instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u) (d :: u). ToProduct (f a, f b, f c, f d) f '[a, b, c, d] where
   |                                     ^

src/Data/WorldPeace/Product.hs:71:46: error:
    Not in scope: type variable ‘u’
   |
71 | instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u) (d :: u). ToProduct (f a, f b, f c, f d) f '[a, b, c, d] where
   |                                              ^

src/Data/WorldPeace/Product.hs:71:55: error:
    Not in scope: type variable ‘u’
   |
71 | instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u) (d :: u). ToProduct (f a, f b, f c, f d) f '[a, b, c, d] where
   |                                                       ^

src/Data/WorldPeace/Product.hs:71:64: error:
    Not in scope: type variable ‘u’
   |
71 | instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u) (d :: u). ToProduct (f a, f b, f c, f d) f '[a, b, c, d] where
   |                                                                ^
```

--- a/src/Data/WorldPeace/Product.hs
+++ b/src/Data/WorldPeace/Product.hs
@@ -53,22 +53,22 @@ class ToProduct (tuple :: *) (f :: u -> *) (as :: [u]) | f as -> tuple where
   toProduct :: tuple -> Product f as
 
 -- | Convert a single value into a 'Product'.
-instance forall (f :: u -> *) (a :: u). ToProduct (f a) f '[a] where
+instance forall u (f :: u -> *) (a :: u). ToProduct (f a) f '[a] where
   toProduct :: f a -> Product f '[a]
   toProduct fa = Cons fa Nil
 
 -- | Convert a tuple into a 'Product'.
-instance forall (f :: u -> *) (a :: u) (b :: u). ToProduct (f a, f b) f '[a, b] where
+instance forall u (f :: u -> *) (a :: u) (b :: u). ToProduct (f a, f b) f '[a, b] where
   toProduct :: (f a, f b) -> Product f '[a, b]
   toProduct (fa, fb) = Cons fa $ Cons fb Nil
 
 -- | Convert a 3-tuple into a 'Product'.
-instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u). ToProduct (f a, f b, f c) f '[a, b, c] where
+instance forall u (f :: u -> *) (a :: u) (b :: u) (c :: u). ToProduct (f a, f b, f c) f '[a, b, c] where
   toProduct :: (f a, f b, f c) -> Product f '[a, b, c]
   toProduct (fa, fb, fc) = Cons fa $ Cons fb $ Cons fc Nil
 
 -- | Convert a 4-tuple into a 'Product'.
-instance forall (f :: u -> *) (a :: u) (b :: u) (c :: u) (d :: u). ToProduct (f a, f b, f c, f d) f '[a, b, c, d] where
+instance forall u (f :: u -> *) (a :: u) (b :: u) (c :: u) (d :: u). ToProduct (f a, f b, f c, f d) f '[a, b, c, d] where
   toProduct :: (f a, f b, f c, f d) -> Product f '[a, b, c, d]
   toProduct (fa, fb, fc, fd) = Cons fa $ Cons fb $ Cons fc $ Cons fd Nil
 
@@ -150,4 +150,3 @@ instance (Show (f a), Show (Product f as)) => Show (Product f (a ': as)) where
   showsPrec :: Int -> (Product f (a ': as)) -> String -> String
   showsPrec n (Cons fa prod) = showParen (n > 10) $
     showString "Cons " . showsPrec 11 fa . showString " " . showsPrec 11 prod
-

